<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

    <arg name="robotID" default="1"/>
    <arg name="namespace" default="machine_$(arg robotID)"/>
    <arg name="numRobots" default="3"/>

    <arg name="projected_objects_array_topic" default="object_detections/projected_to_world_array"/>
    <arg name="target_array_topic" default="MTT/target_array"/>
    <arg name="target_topic" default="MTT/target"/>
    <arg name="offset_topic" default="MTT/offset"/>

    <group ns="$(arg namespace)">
        <arg name="name" value="multi_target_tracker"/>
        <node name="$(arg name)" pkg="multi_target_tracking" type="JPDA_node.py" output="screen">

            <param name="robotID" value="$(arg robotID)" type="int"/>
            <param name="numRobots" value="$(arg numRobots)" type="int"/>

            <param name="projected_objects_array_topic" value="$(arg projected_objects_array_topic)" type="string"/>
            <param name="target_array_topic" value="$(arg target_array_topic)" type="string"/>
            <param name="target_topic" value="$(arg target_topic)" type="string"/>
            <param name="offset_topic" value="$(arg offset_topic)" type="string"/>
        </node>
    </group>

</launch>
